.github/workflows/test.yml

```
@@ -249,11 +249,26 @@ jobs:
           cp -v target/debug/build/onnxruntime-sys-*/out/onnxruntime_*/onnxruntime-*/lib/onnxruntime.dll . || true
           cp -v target/debug/build/onnxruntime-sys-*/out/onnxruntime_*/onnxruntime-*/lib/libonnxruntime.so.* . || true
           cp -v target/debug/build/onnxruntime-sys-*/out/onnxruntime_*/onnxruntime-*/lib/libonnxruntime.*.dylib . || true
-      - name: '`maturin develop`でインストールした`voicevox_core_python_api`を実行'
+      - name: "`maturin develop`でインストールした`voicevox_core_python_api`を実行"
         shell: python
         run: |
           import voicevox_core
           print(voicevox_core)
+      - name: "pytestを実行"
+        shell: bash
+        run: |
+          if [ "${{ matrix.os }}" = "windows-latest" ]; then
+            ORT_PATH="$(find ~+ -name onnxruntime.dll -maxdepth 1 -print -quit)"
+          elif [ "${{ matrix.os }}" = "macos-latest" ]; then
+            ORT_PATH="$(find ~+ -name "libonnxruntime.*.dylib" -maxdepth 1 -print -quit)"
+          else
+            ORT_PATH="$(find ~+ -name "libonnxruntime.so.*" -maxdepth 1 -print -quit)"
+          fi
+          export ORT_PATH
+          echo "ORT_PATH: $ORT_PATH"
+          cd crates/voicevox_core_python_api
+          pip install -r requirements-test.txt
+          pytest
```

上でカレントディレクトリにdll持ってきてここで環境変数に突っ込んでますね。複雑度を下げられそうな気がします！
`cd crates/voicevox_core_python_api`をやめ、`ORT_PATH`をpythonコードとここから省くのはどうでしょう。

```suggestion
      - name: "pytestを実行"
        shell: bash
        run: |
          target_dir="crates/voicevox_core_python_api/"
          pip install -r $target_dir/requirements-test.txt
          PYTHONPATH="$target_dir" pytest $target_dir
```

（`PYTHONPATH="$target_dir"`はなくても動くかも）

---

python_api内にdll持ってくるようにしました。